tests/features/sites_create.feature

```diff
@@ -8,11 +8,9 @@ Feature: Create a site
 
   @vcr sites_create
   Scenario: Create Site
-    When I run "terminus sites create --site=[[test_site_name]] --label=[[test_site_name]] --upstream=WordPress"
-    Then I should get: "Creating new site ..."
+    When I run "terminus sites:create [[test_site_name]] [[test_site_name]] --upstream=e8fe8550-1ab9-4964-8838-2b9abdccf4bf"
```

Comment:
There is no need to alter the 0.x feature test. Just move the inactive 1.x test from the newfeatures dir into the active_features dir.